pyaedt/edb_core/net_class.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import absolute_import  # noreorder
+
+from pyaedt.edb_core.edb_data.nets_data import EDBDifferentialPairData
+from pyaedt.edb_core.edb_data.nets_data import EDBExtendedNetData
+from pyaedt.edb_core.edb_data.nets_data import EDBNetsData
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+class EdbCommon:
+    def __init__(self, pedb):
+        self._pedb = pedb
+
+    @property
+    def _edb(self):
+        """ """
+        return self._pedb.edb_api
+
+    @property
+    def _active_layout(self):
+        """ """
+        return self._pedb.active_layout
+
+    @property
+    def _layout(self):
+        """ """
+        return self._pedb.layout
+
+    @property
+    def _cell(self):
+        """ """
+        return self._pedb.cell
+
+    @property
+    def db(self):
+        """Db object."""
```

Comment:
```suggestion
        """DB object."""
```
Edit assumes you are using "DB" for database? It should probably be spelled out for clarity.